README.es-ES.md

```
@@ -0,0 +1,84 @@
+Si no tienes una máquina AARCH64, puedes usar un servidor de AWS para construir las imágenes.
+
+1. crea un servidor de AWS con las siguientes especificaciones:
+- t4g.[tamaño de instancia]
+- 80GB de almacenamiento
+- imagen de nixOS AARCH64, consulta las amis en [Ami-list](https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/virtualisation/amazon-ec2-amis.nix)
+- acceso SSH, con una llave a la que tengas acceso (con el puerto 22 abierto en tu grupo de seguridad)
+
+> Esta es la configuración que utilicé: [configuration.nix](./configuration.nix)
+
+2. accede por ssh a tu servidor
+3. sigue los pasos de [instalación](#installation)
```

markdownlint-cli2 (0.23.0)

[warning] Link fragments should be valid

(MD051, link-fragments)